hybrid/infrastructure/kubernetes/install.sh

```diff
@@ -5,9 +5,10 @@ set -e
 
 SCRIPT_PATH=$(cd $(dirname $0);pwd)
 
-kubectl create -f ${SCRIPT_PATH}/namespace.yaml
+kubectl apply -f ${SCRIPT_PATH}/namespace.yaml
+kubectl label namespace servicecomb sidecar-injector=enabled --overwrite
```

Comment:
Inject wrong namespace! the install script is only deploy servicecomb components